server.go

```diff
@@ -3936,14 +3936,18 @@ func (s *server) fetchLastChanUpdate() func(lnwire.ShortChannelID) (
 func (s *server) applyChannelUpdate(update *lnwire.ChannelUpdate, outpoint wire.OutPoint,
 	disabled bool) error {
 
-	// Apply the channel update to the relevant links in the switch. This helps keep the
+	// Apply the channel update to the outgoing links in the switch. This helps keep the
 	// switch in sync with certain policy actions like manual disabling of channel.
 	//
 	// NOTE: disabling a channel only disables outgoing payments, not incoming.
-	_, _, edgePolicy, err := s.graphDB.FetchChannelEdgesByOutpoint(&outpoint)
+	var edgePolicy *channeldb.ChannelEdgePolicy
+	_, edgePolicy, edgePolicy2, err := s.graphDB.FetchChannelEdgesByOutpoint(&outpoint)
```

Comment:
Looks good now.
Leaving the same suggestion here as I sent offline. Though I'm fine with leaving it as is for now since this is pretty well covered with an itest.

We could move this logic into a new `FetchOutgoingChannelEdge(ourNodeKey *btcec.PublicKey)` method to the `ChannelGraph` that could be unit tested.